docs/conf.py

```diff
@@ -36,6 +36,7 @@
     'https://groups.google.com/a/learningequality.org/forum/#!forum/dev',
     'http://127.0.0.1:8080',
     'http://127.0.0.1:8080/',
+    'https://www.kiwix.org',
```

Comment:
was this check failing?

Reply:
Yes, for some reason the `make lint` command was repeatedly stumbling on it... 😒